CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Master
+
+##### Enhancements
+
+* Optimize `NSRange` operation.  
+  [Norio Nomura](https://github.com/norio-nomura)
+  [#120](https://github.com/jpsim/SourceKitten/pull/120)
```

Comment:
Please refer to the issues this addresses (#119) rather than this PR number (as explained in https://github.com/jpsim/SourceKitten/blob/master/CONTRIBUTING.md).